tools/system_libs.py

```diff
@@ -528,11 +528,6 @@ def vary_on(cls):
       vary_on += ['is_asan']
     return vary_on
 
-  @classmethod
```

Comment:
was this just dead code?

Reply:
This is the code that used to avoid building versions that are both multithreaded and address sanitized.